devito/finite_differences/derivative.py

```diff
@@ -9,13 +9,13 @@
 from .tools import direct, transpose
 from .rsfd import d45
 from devito.tools import (as_mapper, as_tuple, filter_ordered, frozendict, is_integer,
-                          Reconstructable)
+                          Pickable)
 from devito.types.utils import DimensionTuple
 
 __all__ = ['Derivative']
 
 
-class Derivative(sympy.Derivative, Differentiable, Reconstructable):
+class Derivative(sympy.Derivative, Differentiable, Pickable):
```

Comment:
why isn't Reconstructable enough ?

Reply:
You only get sympy's rkwargs at pickle if you only use Reconstructable so you would loose all the x0/weights/side/transpose at pickle, quite weird it didn't trigger a bug before

Reply:
I wonder whether it's due to this , which we *might* want to lift in Reconstructable. Because as for the rest a Pickable should reconstruct like a Reconstructable...

anyway, no need to investigate now